src/components/layout/Grid/Grid.tsx

```diff
@@ -75,13 +75,26 @@ const Grid: React.FC<GridProps> = ({
   wrapperEl,
   wrapperOverflow = 'hidden',
   ...props
-}) => (
-  <GridWrapper $overflow={wrapperOverflow} as={wrapperEl}>
-    <GridParent $align={align} $cols={cols} $gap={gap} as={parentEl} {...props}>
-      {children}
-    </GridParent>
-  </GridWrapper>
-);
+}) => {
+  if (cols === 1) {
```

Comment:
I'd probably use `less than 2` instead of `equals 1`

Reply:
cols < 2 breaks Storybook, so keeping it as it is, or until some creative person starts using cols={0}